README.md

```diff
@@ -15,6 +15,17 @@ Install dependencies and run the development server:
     $ yarn install
     $ yarn start
 
+### Visual regression testing
+
+The repository includes [BackstopJS](https://github.com/garris/BackstopJS) to test visual changes to the site by comparing a set of reference images for different screen sizes. Anytime the CSS styles are changed, use BackstopJS to test locally:
+
+1. Run the BackstopJS tests: `yarn backstop-test`.
```

Comment:
Do we need to note that the development server needs to be running in another terminal using `yarn start`?